java/src/test/java/com/saucelabs/saucebindings/acceptance/SauceEmuSimAppTest.java

```diff
@@ -0,0 +1,59 @@
+package com.saucelabs.saucebindings.acceptance;
+
+import com.saucelabs.saucebindings.DataCenter;
+import com.saucelabs.saucebindings.SauceMobileOptions;
+import com.saucelabs.saucebindings.SauceMobileSession;
+import io.appium.java_client.AppiumDriver;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.MutableCapabilities;
+
+import static org.junit.Assert.assertTrue;
+
+// TODO: Convert these into unit tests; Integration tests are not unique
+public class SauceEmuSimAppTest {
+    private SauceMobileOptions options;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'options' could be replaced by a local variable.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=5346038)